main.go

```diff
@@ -285,14 +290,18 @@ func main() {
     <h2>Recent Probes</h2>
     <table border='1'><tr><th>Module</th><th>Target</th><th>Result</th><th>Debug</th>`))
 
-		results := rh.List()
+		results := eh.List()
```

Comment:
You're not merging the lists, it should be ordered by time.

Reply:
result struct has no datetime field, so it will be hard to merge and sort by time.

Reply:
You could use the ids

Reply:
The ids in two list might conflicts since both set and increase independently .

Reply:
A common counter will handle that